src/linter/messages.ts

```diff
@@ -333,6 +334,15 @@ export const MESSAGE_INFO = {
 		details: () => `"{@link sap.ui.core.RenderManager#methods/icon RenderManager}",`,
 	},
 
+	[MESSAGE.NOT_STATIC_CONTROL_RENDERER]: {
+		severity: LintMessageSeverity.Warning,
+		ruleId: RULES["no-deprecated-api"],
+
+		message: ({className}: {className?: string}) =>
+			`The control renderer ${className ? ("of '" + className + "'") : ""} must be a static object`,
```

Comment:
I would rather say "...must be a static __property__". Otherwise, one might think that it must be an embedded renderer. But for sure, references are also okay.

Reply:
Thanks!